packages/dashboard-grid-layout/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@uifabric/dashboard-grid-layout",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@microsoft/load-themed-styles": "^1.7.13",
+    "office-ui-fabric-react": ">=6.13.2 <7.0.0",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1"
+  },
+  "main": "lib/index.js",
+  "typings": "lib-es2015/index.d.ts",
+  "scripts": {
+    "build": "node ../../scripts/build.js",
+    "clean": "node ../../scripts/clean.js",
+    "start": "node ../../scripts/start.js",
+    "test": "react-scripts-ts test --env=jsdom",
+    "eject": "react-scripts-ts eject",
+    "start-test": "node ../../scripts/start-test.js",
+    "update-snapshots": "node ../../scripts/build.js jest -u"
+  },
+  "devDependencies": {
+    "@types/jest": "23.0.0",
+    "@types/react": "16.3.16",
+    "@types/react-dom": "16.0.5",
+    "@types/react-grid-layout": "^0.16.4",
+    "@types/webpack-env": "1.13.0",
+    "@uifabric/example-app-base": ">=6.2.1 <7.0.0",
+    "@uifabric/experiments": ">=6.0.0 <7.0.0",
+    "office-ui-fabric-react-tslint": ">=6.0.0 <7.0.0",
+    "react": ">=16.3.2-0 <17.0.0",
```

Comment:
these are dependencies, not devDependencies:

react, react-dom, react-grid-layout.

Also, do you need experiments as a dependency?

Reply:
Yeah I need to show the Card component with an example inside the dashboard grid layout

Reply:
I am thinking would be a good idea for us to move the card here? Because the RGL and card have to go hand in hand.

@dzearing@microsoft.com your thoughts?

---
In reply to: [196641900](https://github.com/OfficeDev/office-ui-fabric-react/pull/5259#discussion_r196641900) [](ancestors = 196641900)